src/components/learner-credit-management/BudgetAssignmentsTable.jsx

```diff
@@ -104,13 +112,13 @@ const BudgetAssignmentsTable = ({
         filters: [],
       }}
       fetchData={fetchTableData}
-      data={tableData.results || []}
+      data={tableData?.results.filter(assignment => assignment.state !== 'cancelled') || []}
```

Comment:
filtering for the state here because the `params` defined here isn't working as expected. https://github.com/openedx/frontend-app-admin-portal/blob/master/src/data/services/EnterpriseAccessApiService.js#L155